docs/source-app/workflows/mount_cloud_object_store.rst

```diff
@@ -0,0 +1,150 @@
+:orphan:
+
+################
+Mount Cloud Data
+################
+
+**Audience:** Users who want to read files stored in a Cloud Object Bucket in an app.
+
+******************************
+Mounting Public AWS S3 Buckets
+******************************
+
+===================
+Add Mount to a Work
+===================
+
+To mount data from a cloud bucket to your app compute, initialize a :class:`~lightning_app.storage.mount.Mount`
+object with the source path of the s3 bucket and the absolute directory path where it should be mounted and
+pass the :class:`~lightning_app.storage.mount.Mount` to the :class:`~lightning_app.utilities.packaging.cloud_compute.CloudCompute`
+of the :class:`~lightning_app.core.work.LightningWork` it should be mounted on.
+
+In this example, we will mount an S3 bucket: ``s3://ryft-public-sample-data/esRedditJson/`` to ``/content/esRedditJson/``.
+
+.. code:: python
+    :emphasize-lines: 9-12
+
+    import lightning as L
+    from lightning_app import CloudCompute
+    from lightning_app.storage import Mount
+
+    class Flow(L.LightningFlow):
+       def __init__(self):
+           super().__init__()
+           self.my_work = MyWorkClass(
+               cloud_compute=CloudCompute(
+                   mounts=Mount(
+                       source="s3://ryft-public-sample-data/esRedditJson/",
+                       mount_path="/content/esRedditJson/",
```

Comment:
can this path be relative?

Reply:
nope. it needs to be absolute. an error is raised in the code if you don't

Reply:
Can that be explicitly mentioned somewhere?

Reply:
This should be the parameter autodoc too